src/Modules/CalcPerform.lua

```diff
@@ -3014,7 +3014,7 @@ function calcs.perform(env, avoidCache)
 				end
 				override = m_max(override, effect or 0)
 			end
-			output["Maximum"..ailment] = modDB:Override(nil, ailment.."Max") or ailmentData[ailment].max
+			output["Maximum"..ailment] = modDB:Override(nil, ailment.."Max") or (modDB:Sum("BASE", nil, ailment.."Max") and (ailmentData[ailment].max + modDB:Sum("BASE", nil, ailment.."Max"))) or ailmentData[ailment].max
```

Comment:
```suggestion
			output["Maximum"..ailment] = modDB:Override(nil, ailment.."Max") or (ailmentData[ailment].max + modDB:Sum("BASE", nil, ailment.."Max"))
```
Guess it will be fine as it returns 0 by default